osf/features.py

```diff
@@ -8,6 +8,9 @@
 OSF_PREREGISTRATION = 'osf_preregistration'
 OSF_GROUPS = 'osf_groups'
 ENABLE_CHRONOS = 'enable_chronos'
+SLOAN_COI = 'waffle:sloan|coi'
```

Comment:
As mentioned above, these should not be used for the validation section. Instead, we should have an additional **switch** (not flag, and we can add each per PR) for the whole functionality. The front end will use that to know whether to give the edit controls for the users, and we can use it to regulate the above validation.